keras_nlp/models/bert/bert_masked_lm_preprocessor.py

```diff
@@ -134,29 +134,43 @@ def __init__(
             truncate=truncate,
             **kwargs,
         )
+        self.mask_selection_rate = mask_selection_rate
+        self.mask_selection_length = mask_selection_length
+        self.mask_token_rate = mask_token_rate
+        self.random_token_rate = random_token_rate
 
+        self.masker = None
+
+
+    def build(self, input_shape):
+        # Defer masker creation to `build()` so that we can be sure tokenizer
+        # assets have loaded when restoring a saved model.
+        super().build(input_shape)
```

Comment:
I don't think we should need to chain to super here, or guard against multiple `build` calls in succession. At least most of our layers do not https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/modeling/position_embedding.py#L95-L103

So we can just do
```
self.masker = ...
self.built = True
```

Reply:
This one will need it to create the packer, but at the level above this (preprocessor) I will not to a chain to super.